macros/utils/helpers/parse_column.sql

```diff
@@ -0,0 +1,11 @@
+{% macro parse_column(table_alias, column) %}
+
+{% set columns = dbt_activity_schema.columns() %}
+{%- if column not in columns.values() -%}
+    {%- set parsed_column = dbt_activity_schema.json_unpack_key(table_alias ~ '.' ~ columns.feature_json, column) -%}
```

Comment:
It's a nice convenience feature, and we may be able to extend it in the future if we formally define types of packed columns.